builds/mariner/SPECS/azure-iotedge/azure-iotedge.spec

```diff
@@ -0,0 +1,186 @@
+Summary:        Azure IoT Edge Security Daemon
+Name:           azure-iotedge
+Version:        1.0.10.4
+Release:        1%{?dist}
+
+# A buildable azure-iotedge environments needs functioning submodules that do not work from the archive download
+# To recreate the tar.gz run the following
+#  sudo git clone https://github.com/Azure/iotedge.git -b %{version}
+#  pushd iotedge
+#  sudo git submodule update --init --recursive
+#  popd
+#  sudo mv iotedge azure-iotedge-%{version}
+#  sudo tar -cvf azure-iotedge-%{version}.tar.gz azure-iotedge-%{version}/
+
+#Source0:       https://github.com/Azure/iotedge/archive/%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+License:        MIT
+Group:          Applications/File
+URL:            https://github.com/azure/iotedge
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+BuildRequires:  rust >= 1.40.0
```

Comment:
This should be an `=` relation on the exact version listed in `edgelet/rust-toolchain`, otherwise the Mariner packages will be built with a different toolchain than every other package, which is the problem we've already talked about.